### ISSUE TITLE
Error "content = line["choices"][0]["delta"].get("content")" in Docker Compose

### ISSUE THREAD

**Issue by seth100** [xtekky/gpt4free#1273]
**Bug description**

Following [these steps](https://github.com/xtekky/gpt4free#setting-up-with-docker) to use it via Docker Compose, has always worked, but today, after pulling latest changes from repo, I got:

```sh
gpt4free-gui  | SyntaxError: invalid syntax
gpt4free-gui  | Traceback (most recent call last):
gpt4free-gui  |   File "/usr/local/lib/python3.9/runpy.py", line 188, in _run_module_as_main
gpt4free-gui  |     mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
gpt4free-gui  |   File "/usr/local/lib/python3.9/runpy.py", line 111, in _get_module_details
gpt4free-gui  |     __import__(pkg_name)
gpt4free-gui  |   File "/app/g4f/__init__.py", line 3, in <module>
gpt4free-gui  |     from .models    import Model, ModelUtils, _all_models
gpt4free-gui  |   File "/app/g4f/models.py", line 4, in <module>
gpt4free-gui  |     from .Provider   import BaseProvider, RetryProvider
gpt4free-gui  |   File "/app/g4f/Provider/__init__.py", line 27, in <module>
gpt4free-gui  |     from .GptGo           import GptGo
gpt4free-gui  |   File "/app/g4f/Provider/GptGo.py", line 66
gpt4free-gui  |     content = line["choices"][0]["delta"].get("content"):
```

**Environment**
- Docker v24.0.7
- Docker Compose v2.21.0
- Ubuntu 22.04

**Comment by Lin-jun-xiang** [comment]
@seth100 this will be fixed when PR #1272 be merged.

**Comment by xtekky** [comment]
apologise for this mistake, but the pypi version should be fine ; )
